### DISCUSSION TITLE
Is this library still being maintained?

### DISCUSSION THREAD

**Discussion by stevengunneweg** [easepick/easepick#266] [Q&A]
Hello there,

I'm currently using the old Litepicker library for a project and was looking into upgrading to Easepick.
Easepick looks great and the demo works beautifully however when I look at the commits, this repository seems stale since the last commit on 23 Feb 2023.

Is there an active fork or should I consider simply using another library instead that is actually being maintained?

**Comment by kevinkimball** [comment]
Also interested to know if there's any active forks. Quite a few feature requests and PRs. Love the library

**Comment by SrimanNarayanaSangha** [comment]
Love this library also. Minimalist and answer most needs + extandable, very neat!
